File: Beecrowd/1436 Jogo do Tijolo/main.c
#include <stdio.h>

int main(){
    int t, n, idade[12], i, met, cont = 1;

    scanf("%d", &t);

    while(t--){
        scanf("%d", &n);
        for(i = 0; i < n; i++)
            scanf("%d", &idade[i]);

        met = n / 2;

        printf("Case %d: %d\n", cont++, idade[met]);
    }

    return 0;
}

File: Beecrowd/1436 Jogo do Tijolo/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);
        final int TAM = 12;

        int t, n, met, cont = 1;

        int[] idade = new int[TAM];

        t = sc.nextInt();

        while (t > 0) {
            n = sc.nextInt();

            for (int i = 0; i < n; i++)
                idade[i] = sc.nextInt();

            met = n / 2;

            System.out.println("Case " + (cont++) + ": " + idade[met]);

            t--;
        }

        sc.close();
    }

}